Description: Configure linking for C-library wrapping modules
 Use the system C libraries, rather than sources bundled with cPython, or
 anything from /usr/local.
 This is for performance, to avoid unnecessary C extension imports at start-up.

Forwarded: not-needed

--- a/Modules/Setup
+++ b/Modules/Setup
@@ -109,6 +109,7 @@ TESTPATH=
 COREPYTHONPATH=$(DESTPATH)$(SITEPATH)$(TESTPATH)
 PYTHONPATH=$(COREPYTHONPATH)
 
+#_hashlib _hashopenssl.c -lssl -lcrypto
 
 # ---
 # Built-in modules required to get a functioning interpreter are listed in
@@ -161,8 +162,8 @@ PYTHONPATH=$(COREPYTHONPATH)
 #_sysconfig _sysconfig.c
 
 # XML
-#_elementtree _elementtree.c
-#pyexpat pyexpat.c
+#_elementtree _elementtree.c -lexpat
+#pyexpat pyexpat.c -lexpat
 
 # hashing builtins
 #_blake2 _blake2/blake2module.c _blake2/blake2b_impl.c _blake2/blake2s_impl.c
@@ -207,15 +208,8 @@ PYTHONPATH=$(COREPYTHONPATH)
 #readline readline.c -lreadline -ltermcap
 
 # OpenSSL bindings
-#_ssl _ssl.c $(OPENSSL_INCLUDES) $(OPENSSL_LDFLAGS) $(OPENSSL_LIBS)
-#_hashlib _hashopenssl.c $(OPENSSL_INCLUDES) $(OPENSSL_LDFLAGS) -lcrypto
-
-# To statically link OpenSSL:
-# _ssl _ssl.c $(OPENSSL_INCLUDES) $(OPENSSL_LDFLAGS) \
-#     -l:libssl.a -Wl,--exclude-libs,libssl.a \
-#     -l:libcrypto.a -Wl,--exclude-libs,libcrypto.a
-# _hashlib _hashopenssl.c $(OPENSSL_INCLUDES) $(OPENSSL_LDFLAGS) \
-#     -l:libcrypto.a -Wl,--exclude-libs,libcrypto.a
+#_ssl _ssl.c -lssl -lcrypto
+#_hashlib _hashopenssl.c -lcrypto
 
 # The _tkinter module.
 #
